Version 1.35 produces error during start up on Beta environment

teamengine-beta of teamengine-docker project can be used to reproduce the behavior.

Following error occurs during start up of TEAM Engine:
```
SEVERE: ContainerBase.addChild: start: 
org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/te2]]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:162)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:1018)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:994)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:652)
	at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1296)
	at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:2038)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    com/sun/jersey/server/impl/container/servlet/JerseyServletContainerInitializer.addServletWithDefaultConfiguration(Ljavax/servlet/ServletContext;Ljava/util/Set;)V @46: invokespecial
  Reason:
    Type 'com/sun/jersey/api/core/DefaultResourceConfig' (current frame, stack[2]) is not assignable to 'javax/ws/rs/core/Application'
  Current Frame:
    bci: @46
    flags: { }
    locals: { 'com/sun/jersey/server/impl/container/servlet/JerseyServletContainerInitializer', 'javax/servlet/ServletContext', 'java/util/Set', 'javax/servlet/ServletRegistration', 'java/util/Set' }
    stack: { uninitialized 33, uninitialized 33, 'com/sun/jersey/api/core/DefaultResourceConfig' }
  Bytecode:
    0x0000000: 2b13 0020 b600 0ab9 000b 0200 4e2d c600
    0x0000010: 902d b900 1d01 00c7 0087 2a2c b700 213a
    0x0000020: 04bb 0022 59bb 0023 5919 04b7 0024 b700
    0x0000030: 253a 052b 2db9 0026 0100 1905 b900 2703
    0x0000040: 004e 2db9 0028 0100 b900 2901 0099 0029
    0x0000050: b200 2abb 002b 59b7 002c 122d b600 2e2d
    0x0000060: b900 2601 00b6 002e 122f b600 2eb6 0030
    0x0000070: b600 31a7 002b b200 2abb 002b 59b7 002c
    0x0000080: 1232 b600 2e2d b900 2601 00b6 002e 1233
    0x0000090: b600 2e19 04b6 0034 b600 30b6 0035 b1  
  Stackmap Table:
    append_frame(@118,Object[#130],Object[#146],Object[#147])
    chop_frame(@158,2)

	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.apache.catalina.startup.WebappServiceLoader.loadServices(WebappServiceLoader.java:197)
	at org.apache.catalina.startup.WebappServiceLoader.load(WebappServiceLoader.java:158)
	at org.apache.catalina.startup.ContextConfig.processServletContainerInitializers(ContextConfig.java:1579)
	at org.apache.catalina.startup.ContextConfig.webConfig(ContextConfig.java:1273)
	at org.apache.catalina.startup.ContextConfig.configureStart(ContextConfig.java:881)
	at org.apache.catalina.startup.ContextConfig.lifecycleEvent(ContextConfig.java:388)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:117)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:90)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5606)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:145)
	... 10 more

May 31, 2023 4:50:33 PM org.apache.catalina.startup.HostConfig deployDirectory
SEVERE: Error deploying web application directory /usr/local/tomcat/webapps/te2
java.lang.IllegalStateException: ContainerBase.addChild: start: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/te2]]
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:1022)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:994)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:652)
	at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1296)
	at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:2038)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
```

Might be related to https://github.com/opengeospatial/ets-wfs20-dgiwg/issues/23.

--- a/src/assembly/deps.xml
+++ b/src/assembly/deps.xml
@@ -22,6 +22,8 @@
         <exclude>net.sourceforge.saxon:saxon9*</exclude>
         <exclude>xml-resolver:xml-resolver</exclude>
         <exclude>com.thaiopensource:jing</exclude>
+        <exclude>javax.ws.rs:jsr311-api</exclude>
+        <exclude>com.beust:jcommander</exclude>
       </excludes>
       <useTransitiveDependencies>true</useTransitiveDependencies>
       <useTransitiveFiltering>true</useTransitiveFiltering>

